lib/registries/addon/my-registrations/template.hbs

```diff
@@ -15,22 +17,61 @@
             {{#if this.currentUser.user}}
                 <BsTab
                     local-class='NavTabs'
+                    @customTabs={{true}}
                     @onChange={{this.changeTab}} 
                     @activeId={{this.tab}}
                     as |tab|
                 >
-                    <div local-class='SortDescription'>
+                    <BsNav
+                        @type='tabs'
+                        as |nav|
+                    >
+                        <nav.item
+                            data-test-my-registrations-nav='drafts'
+                            @active={{bs-eq tab.activeId 'drafts'}}
+                        >
+                            <button
+                                data-test-my-registrations-nav-button='drafts'
+                                local-class='NavItem'
+                                type='button'
+                                role='tab'
+                                {{on 'click' (fn this.changeTab 'drafts')}}
+                            >
+                                {{t 'registries.my_registrations.drafts'}}
+                            </button>
+                        </nav.item>
+                        <nav.item
+                            data-test-my-registrations-nav='submitted'
+                            @active={{bs-eq tab.activeId 'submitted'}}
+                        >
+                            <button
+                                data-test-my-registrations-nav-button='submitted'
+                                local-class='NavItem'
+                                type='button'
+                                role='tab'
+                                {{on 'click' (fn this.changeTab 'submitted')}}
```

Comment:
You mentioned mobile earlier. Do we include any event handlers in the OSF codebase for mobile interaction? Very general, but in regards to this: https://developer.mozilla.org/en-US/docs/Web/API/Touch_events/Using_Touch_Events

Reply:
Mobile devices will send `click` event after touch events. So if you don't need to distinguish between `touchstart`, `touchmove` and `touchend`, it is okay to just bind to the `click` event.

Reply:
Ok, the repo found [here](https://github.com/kaimallea/isMobile/blob/master/src/isMobile.ts) is pretty honest about how it's checking the browser/device OS and handing back event data. Either installing a dependency like this or including our own but similar code could improve overall UX.

Reply:
From my limited experience working with touch events, the differences between operating systems and machines was such a pain to handle. I think having a dependency handle that logic might not be bad, but I think [pointer events](https://developer.mozilla.org/en-US/docs/Web/API/Pointer_events) may be the best approach, as this is web API is supposed to be hardware agnostic.